programs/bpf/c/src/return_data/return_data.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @brief return data Syscall test
+ */
+#include <solana_sdk.h>
+
+#define DATA "the quick brown fox jumps over the lazy dog"
+
+extern uint64_t entrypoint(const uint8_t *input) {
+  uint8_t buf[1024];
+
+  // There should be no return data on entry
+  uint64_t ret = sol_get_return_data(NULL, 0);
+
+  sol_assert(ret == 0);
+
+  // set some return data
+  sol_set_return_data((const uint8_t*)DATA, sizeof(DATA));
+
```

Comment:
How about adding this pattern to the test after the data is set, I image folks will use it to get the length of the return value:
```rust
int64_t num_bytes = sol_get_return_data(NULL, 0);
```

Reply:
Good idea, added.